pixi.toml

```
@@ -53,8 +53,8 @@ initialize-julia = { depends_on = [
 # Docs
 build-julia-docs = { cmd = "julia --project docs/make.jl", depends_on = [
     "initialize-julia",
-] }
-quartodoc-build = { cmd = "quartodoc build && rm objects.json", cwd = "docs" }
+], inputs = ["core"] , outputs = ["docs"]}
+quartodoc-build = { cmd = "quartodoc build && rm objects.json", cwd = "docs" , inputs = ["objects.json"], outputs = ["docs"]}
```

```suggestion
quartodoc-build = { cmd = "quartodoc build && rm objects.json", cwd = "docs" , inputs = ["docs/_quarto.yml", "python/ribasim"], outputs = ["docs/python/reference"]}
```

This builds the API docs of Ribasim Python. So it will depend on Ribasim Python.
Its configuration is in `_quarto.yml`.

The output you can again guess from the URL :)
https://deltares.github.io/Ribasim/python/reference/

---

This is not resolved